### PR TITLE
Fix travis and tools

### PR DESCRIPTION
**Pull request by ale-rt**
#37 made the code black.
This PR wants to make development tools aware of that.
Also do not run tests on travis which will never ever succeed

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+lines_after_imports = 2
+line_length = 88
+multi_line_output=3
+include_trailing_comma=True
+not_skip = __init__.py
 
+[flake8]
+exclude = bootstrap.py,docs,*.egg.,omelette
+max-complexity = 15
+max-line-length = 88
```
Comment (tisto) [inline setup.cfg]: @ale-rt I would set line_length to something like 200 to not to collide with black.

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ env:
     - PLONE_VERSION=5.2.x
     - PLONE_VERSION=5.1.x
 matrix:
-  allow_failures:
+  exclude:
```
Comment (tisto) [inline .travis.yml]: @ale-rt nice. I was looking for that but didn't recall the way to do this.

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,17 @@
-[templer.local]
-template = plone_basic
+[isort]
```
Comment (tisto) [inline setup.cfg]: @ale-rt I guess we kind of have to decide where we want to keep configuration like this. So far some of this is in p.r.codeanalysis. I am ok moving into a more pythonic direction. Though, then we should reflect that in the p.r.codeanalysis docs.

File: .travis.yml (hunk 2 above)
Comment (ale-rt) [reply .travis.yml]: Well, it was already like that on master...

File: setup.cfg (hunk 1 above)
Comment (ale-rt) [reply setup.cfg]: Black is 88

**Comment by ale-rt** [comment]
@tisto I also made setup.py black and ran isort (anyway no test for the imports or for the code to be black is in place yet)

**Comment by tisto** [comment]
@ale-rt yeah. that's a problem in my setup. setup.py is excluded from black. Though, running black on top-level also causes trouble. I am open for suggestions here. :)

I kind of personally gave up on sorting imports because there was always too much discussion around this (like before black and prettier).

File: .travis.yml (hunk 2 above)
Comment (tisto) [reply .travis.yml]: Jikes :/

File: setup.cfg (hunk 1 above)
Comment (tisto) [reply setup.cfg]: Not always if I recall correctly. Do you have a pointer regarding this?

Comment (ale-rt) [reply setup.cfg]: https://github.com/psf/black#line-length

Comment (tisto) [reply setup.cfg]: @ale-rt right, I recall reading that and setting line_length to 88. Though, then at some point black was creating longer lines so we moved to 200.

**Pull request merged**